### PR TITLE
Fixups from Migrator pull request, and CI to GitHub Actions

### PR DESCRIPTION
**Pull request by markstory**
* Do a few fixups for #488 
* Port CI configuration to GitHub Actions as travis is on the way out.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    branches:
+      - '*'
+
+jobs:
+  testsuite-linux:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.2', '7.4']
```
Comment (dereuromark) [inline .github/workflows/ci.yml:18]:
```suggestion
        php-version: ['7.2', '8 0']
```

File: src/TestSuite/Migrator.php [hunk 2]
```diff
@@ -22,18 +21,13 @@
...
-
     /**
      * General command to run before your tests run
      * E.g. in tests/bootstrap.php
      *
-     * @param array $config
+     * @param array $config Configuration data
      * @param bool  $verbose Set to true to display messages
-     * @return Migrator
+     * @return self
```
Comment (dereuromark) [inline src/TestSuite/Migrator.php:30]: Static?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (markstory) [reply .github/workflows/ci.yml:18]: I don't know if we've done the work to get migrations working on 8.0. I can add that to the matrix once tests are passing again.

**Comment by dereuromark** [comment]
I will get this merged, as we can do the PHPStan as follow up
And I can check now PHP8 in a PR

**Pull request merged**